BCStudentSoftwareDevTeam/celts
# Date picker 1343

### **_Issue Description_**
********************************************************
Fixes issue #1343
Within the multiple time offerings modal, the time picker previously allowed users to create events in the past without toggling the "Allow start date to be in the past" option.

### **_Changes_**
**************************************************************
- Updated the time picker logic to enforce the "Allow start date to be in the past" toggle.
- Implemented a validation check that only permits event creation in the past when the toggle is enabled.
- Added a flash message when attempting to select a past date without enabling the toggle.

### **_Testing_**
*****************************************************************
- Verified that the time picker correctly restricts past date selections when the "Allow start date to be in the past" option is disabled.
- Conducted tests using the following steps:
- Checkout the branch datePicker_1343
- Run with flask run
- Open the modal and try selecting a past date without enabling the toggle.
- Enable the toggle and confirm that past dates are now selectable.

The issue has been fixed! The multiple time offerings modal now properly respects the past dates toggle.
